src/env.h

```diff
@@ -266,6 +266,13 @@ struct node_ares_task {
   RB_ENTRY(node_ares_task) node;
 };
 
+// 4096 bytes on x64 and 4088 on ia32.
+struct node_destroy_ids_list {
+  int64_t ids[510];
```

Comment:
teeny-tiny nit: Could you make this the last member of the struct so that `length`, `prev` and the first few entries all fit into a single cache line?